lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
Interesting, I'm not sure in that case. Seems like the sequence will need a different retry setting.  Although, I'm not familiar with sequences. Is that the callback type of transaction in sequelize?

In any case, that seems like a job for the next PR.